bundles/org.openhab.binding.bluetooth/README.md

```diff
@@ -50,18 +52,31 @@ Every Bluetooth thing has the following channel:
 | battery_level | Number    | The device's battery level in percent                           |
 
 
+Devices which use a `roaming` adapter as their bridge also gain the following channels:
+
+| Channel ID       | Item Type | Description                                                     |
+|------------------|-----------|-----------------------------------------------------------------|
+| adapter-uid      | String    | The thingUID of the adapter that is nearest to this device      |
+| adapter-location | String    | The value of the `Location` specified for the nearest adapter    |
+
+
+## Roaming Adapter
```

Comment:
Just like the other adapters, this one should go into its own bundle. Is there any strong reason against that?

Reply:
I initially attempted something akin to that approach but I found that the BluetoothDiscoveryService would need to handle roaming bluetooth devices differently than other adapters, and in order to do that they would need awareness of what a roaming was. This awareness, as well as an overall increase in the complexity of the problems that would need to be resolved, made me decide to just embed the roaming adapter into the base bluetooth bundle itself.

This does make sense in a way, since the roaming adapter isn't any sort of concrete implementation of a Bluetooth adapter as it delegates all of its operations to a real adapter instead.

Reply:
The roaming adapter is nonetheless just an optional feature, which could be left away easily and not a mandatory core part of the binding. I think wrt code cleanliness and structure, it would be nice to have it as a separate bundle.
Wrt the discovery service I'd suggest to think about a way to introduce an interface to the "generic" discovery service, which allows the customization of its logic by extensions. Similar to the "UPnPDiscoveryParticipant", which plugs into the "UPnPDiscoveryService".
Do you think that would be feasible?

Reply:
> Wrt the discovery service I'd suggest to think about a way to introduce an interface to the "generic" discovery service, which allows the customization of its logic by extensions. Similar to the "UPnPDiscoveryParticipant", which plugs into the "UPnPDiscoveryService".
Do you think that would be feasible?

I think that should be feasible, I'll get to work on that. Thanks!